data/common.yaml

```diff
@@ -0,0 +1,114 @@
+---
+lookup_options:
+  '^gnome::(d|g)conf_hash$':
+    merge:
+      strategy: deep
+      knockout_prefix: --
+  gnome::packages:
+    merge:
+      strategy: deep
+      knockout_prefix: --
+
+gnome::configure: true
+gnome::packages:
+  gnome-session: ~
+  gnome-session-xsession: ~
+  gnome-settings-daemon: ~
+  gnome-terminal: ~
+  gnome-user-docs: ~
+  im-chooser: ~
+  libgnome: ~
+  libgnomeui: ~
+  nautilus: ~
+  orca: ~
+  yelp: ~
+
+gnome::dconf_profile_hierarchy:
+  - user-db:user
+  - system-db:simp
+  - system-db:local
+  - system-db:site
+  - system-db:distro
+
+gnome::dconf_hash:
+  simp:
+    org/gnome/desktop/media-handling:
+      # Prevent mount points from being automatically created
+      automount:
+        value: false
+      # Prevent automounted devices from opening automatically
+      automount-open:
+        value: false
+      # Prevent automounted devices from executing automatically
+      automount-never:
+        value: false
+    org/gnome/settings-daemon/plugins/media-keys:
```

Comment:
ctl-alt-del is not disabled.  When, as a normal user I select it it will try to power off the machine. Plus a normal user has access to the shutdown button on the screen.